homeassistant/components/flux_led/__init__.py

```diff
@@ -131,7 +131,7 @@ async def _async_sync_time(*args: Any) -> None:
         await device.async_set_time()
 
     await _async_sync_time()  # set at startup
-    entry.async_on_unload(async_track_time_change(hass, _async_sync_time, 2, 4, 30))  # type: ignore[arg-type]
+    entry.async_on_unload(async_track_time_change(hass, _async_sync_time, 2, 4, 30))
```

Comment:
Extra note for `flux_led` code owner: should this be `2, 40, 30` instead of `2, 4, 30`

Reply:
I guess it should be `2:40:30`. At least the docstring says so. Opened #63554
```py
"""Set the time every morning at 02:40:30."""
```